src/tasks/installers/olm.ts

```diff
@@ -417,13 +417,11 @@ export class OLMTasks {
     }
   }
 
-  private getOlmNamespaceLabels(flags: any) {
-    let labels = Object.create({})
-
+  private getOlmNamespaceLabels(flags: any): any {
     //The label values must be strings
-    if (flags.metrics && flags.platform === 'openshift') {
-      labels['openshift.io/cluster-monitoring'] = 'true'
+    if (flags['cluster-monitoring'] && flags.platform === 'openshift') {
```

Comment:
```suggestion
    if (flags['cluster-monitoring'] && ctx.isOpenShift4) {
```

Reply:
This one is good.

Reply:
I'll remove getOlmNamespaceLabels function to common task